### DISCUSSION TITLE
求助关于添加预设emoji表情

### DISCUSSION THREAD

**Discussion by ghost** [walinejs/waline#771] [Q&A] [answered]
我使用的是Butterfly主题，在Vercel 部署，不知道在那个文件里可以直接将预设添加到 emoji ，谢谢

**Comment by elkan1788** [comment, marked as answer]
参考官方文档就可以，在你的引用位置添加个 'emoji' 参数，然后找到链接 [https://waline.js.org/guide/client/emoji.html ](https://waline.js.org/guide/client/emoji.html ) 中支持的预设地址加进去就可以。
